Chess/game.cpp

```diff
@@ -0,0 +1,45 @@
+#include "game.h"
+
+#ifdef DEBUG_LOGS
+#include <iostream>
+#endif // DEBUG_LOGS
+
+Game::Game() {};
+
+
+void Game::start()
+{
+    bool isGameOpen = true;
+
+    std::vector<Player*> players(2);
+
+    players[0] = new HumanPlayer(_WHITE);
+    players[1] = new HumanPlayer(_BLACK);
```

Comment:
Sunt lucruri de adresat ramase in PR-ul vechi
https://github.com/Junior-Coders-Devs/AI-Challenge/pull/42

Reply:
Am facut modificarile la game si aici si pe PR la game. Nu stiu daca mai are rost sa fac un PR nou pentru imagini, ca na is doar imagini :)))